test/integration/servicebus/config.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+module.exports = {
+
```

Comment:
I agree, ish. I think address makes sense, and defaultLink/queueName/whatever makes sense, but beyond that I'm less sure. I'm hoping to write some tests for topics this week - those are different enough to muck up any assumptions about commonalities, I'm hoping.